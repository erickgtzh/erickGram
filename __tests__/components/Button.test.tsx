import Button from '../../src/components/Button/Button';
import {render} from '@testing-library/react-native';

it('should render the Button', () => {
  const {toJSON} = render(<Button />);
  expect(toJSON()).toMatchSnapshot();
});
