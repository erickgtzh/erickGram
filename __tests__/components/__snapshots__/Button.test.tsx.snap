// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Button 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    [
      {
        "alignItems": "center",
        "borderColor": "#dcdcdc",
        "borderRadius": 5,
        "borderWidth": 1,
        "margin": 5,
        "padding": 5,
      },
      {},
    ]
  }
>
  <Text
    style={
      {
        "color": "#000000",
        "fontWeight": "600",
      }
    }
  />
</View>
`;
